common_design_subtheme/site.webmanifest

```
@@ -0,0 +1,20 @@
+{
+    "name": "Common Design Demo",
+    "short_name": "CD",
```

```suggestion
    "name": "YOUR WEBSITE NAME",
    "short_name": "WEBSITE",
```

IMO the values here look a little too "finished," and I'd rather put default values that are plainly in need of attention.